### DISCUSSION TITLE
Do you store historical data?

### DISCUSSION THREAD

**Discussion by tif-calin** [alltheplaces/alltheplaces#5664] [Q&A] [answered]
The weekly run/build is cool and interesting. I think the data generated from historical runs would be really cool to play with to watch as different chains grow/expand/die over time

**Comment by tif-calin** [comment, marked as answer]
Woops, just had to dig through a bit. You can see historical data on the website https://www.alltheplaces.xyz/wikidata with the drop down at the top. And you can download past builds here https://www.alltheplaces.xyz/builds

**Reply by Cj-Malone** [reply]
Yep. Let us know if you have anymore questions.

Some notes you'll probably see when looking at the details:
 - We change spider names occasionally
 - Some spiders fluctuate each week, for various reasons
 - 1 brand can have multiple spiders, usually split by country
 - Spiders break, sometimes for a long time, some days the internet gods are just angry
